x-pack/plugins/security_solution/server/lib/detection_engine/schemas/rule_schemas.ts

```diff
@@ -274,6 +275,7 @@ export const internalRuleCreate = t.type({
   }),
   enabled,
   actions: actionsCamel,
+  responseActions: responseActionsCamel,
```

Comment:
Since we're implementing the response action scheduling as part of the security rule type now rather than as a connector, does it still make sense to keep `responseActions` as a top level parameter? The top level parameters are typically managed/used by the framework, whereas parameters that are used by the solution are defined in `ruleParams` (https://github.com/elastic/kibana/blob/f738d350ae9ca0d2ac27387b322886e616a2cb09/x-pack/plugins/security_solution/server/lib/detection_engine/schemas/rule_schemas.ts#L87-L122 for the shared security rule type params).

Reply:
It got moved to `params`, great feedback, thank you!